src/esn.calendar.libs/app/services/calendar-api.spec.js

```diff
@@ -29,9 +29,9 @@ describe('The calendar module apis', function() {
   }
 
   beforeEach(function() {
-    module('esn.calendar.libs');
+    angular.mock.module('esn.calendar.libs');
```

Comment:
Instead of having to do this everywhere, you can tell webpack to do it for you then you will not have to change all the files